src/applications/mhv/secure-messaging/components/Search/SearchForm.jsx

```diff
@@ -63,6 +66,9 @@ const SearchForm = props => {
   };
 
   const handleSearch = () => {
+    setFiltersCleared(false);
+    if (filterBoxRef.current.checkFormValidity()) return;
```

Comment:
More curious than anything, How did you land on using refs for validation?

Reply:
So I though of using forwardRef as a way to call `checkFormValidity` function from a child component. Sort of the opposite of a callback approach. Here in particular, calling `checkFormValidity` to  see if the FilterBox has all required fields instead of bringing all that one level up to a parent component via callbacks and props